src/components/MemberSearchPage/MemberSearchContainer.js

```diff
@@ -2,97 +2,102 @@ import React, { Component } from 'react';
 import { connect } from 'react-redux';
 import fetchListOfMembersAction from 'components/MemberSearchPage/fetchListOfMembers';
 import { bindActionCreators } from 'redux';
-import { Icon, Input, Tooltip } from 'antd';
+import { Icon, Input, Layout, Tooltip } from 'antd';
 import MemberTable from './MemberTable';
 import PropTypes from 'prop-types';
 import MemberSearchBorder from 'components/MemberSearchPage/MemberSearchBorder';
 import { withTheme } from 'emotion-theming';
+import FilterDrawer from 'components/MemberSearchPage/FilterDrawer';
+import {
+  requestCurrentPageUpdate,
+  requestMemberPerPageUpdate,
+  requestQueryStringUpdate,
+} from 'components/MemberSearchPage/actions';
+import { getCurrentEnd, getCurrentStart, getSelectedFilter } from './utils';
 
 class MemberSearchContainer extends Component {
-  state = {
-    queryString: '',
-    currentPage: 1,
-    membersPerPage: 10,
-  };
-
   static propTypes = {
     pending: PropTypes.bool.isRequired,
     error: PropTypes.object,
     count: PropTypes.object,
     members: PropTypes.arrayOf(PropTypes.object),
   };
 
-  getCurrentStart = (page = this.state.currentPage, pageSize = this.state.membersPerPage) => {
-    return pageSize * (page - 1);
-  };
-
-  getCurrentEnd = (page = this.state.currentPage, pageSize = this.state.membersPerPage) => {
-    return page * pageSize;
-  };
-
   handleChange = e => {
     this.props.fetchListOfMembers(e.target.value, {
-      start: this.getCurrentStart(this.state.currentPage),
-      end: this.getCurrentEnd(this.state.currentPage),
+      start: getCurrentStart(this.props.currentPage, this.props.membersPerPage),
```

Comment:
Personal Taste : I'd use deconstructing => { myVar } = this.props vs this.props.myVar